### ISSUE TITLE
Exception running Flanium with Selenium Grid in parallel

### ISSUE THREAD

**Issue by rhapsodyman** [lanit-exp/FlaNium.Desktop.Driver#15]
flanium driver version: 2.2.0
selenium version: 4.10
selenium-server version: 4.10.0

Unable to execute tests in parallel using Selenium grid.
Having such error messages in selenium hub log when running in parallel on two machines :
```

5:08:53.779 INFO [GridModel.release] - Releasing slot for session id AwesomeSession
15:08:53.780 INFO [LocalSessionMap.lambda$new$0] - Deleted session from local Session Map, Id: AwesomeSession
15:08:54.169 WARN [SeleniumSpanExporter$1.lambda$export$1] - Unable to execute request for an existing session: Unable to find session with ID: AwesomeSession
```

It may be related to the fact that sessionId is not unique and uses same value: "AwesomeSession"

**Comment by cmttwd** [comment]
Hello, edits will be reviewed and added soon

**Comment by cmttwd** [comment]
Fixed in 2.2.1
